src/main/fulcro/client/mutations.cljc

```diff
@@ -174,6 +174,26 @@
   (let [value (ensure-integer (if event (target-value event) value))]
     (set-value! component field value)))
 
+#?(:cljs
+   (defn- ensure-double
+     "Helper for set-integer!, use that instead. It is recommended you use this function only on UI-related
```

Comment:
Might want to update (or remove) docstring

Reply:
I think we can do without the docstring